pyiron_atomistics/atomistics/structure/periodic_table.py

```diff
@@ -89,6 +84,12 @@ def __getitem__(self, item):
         if item in self.sub.index:
             return self.sub[item]
 
+    def get(self, item, default=None):
+        value = self[item]
```

Comment:
Shouldn't this be wrapped in try/except? If the `self[key]` raises an error I would expect `get(key, ANY)` to return `ANY`.